docs/features/web-api-template.md

```diff
@@ -37,6 +37,7 @@ And additional features available with options:
   * `SharedAccessKey`: adds [shared access key authentication](https://webapi.arcus-azure.net/features/security/auth/shared-access-key) mechanism to the API project
   * `Certificate`: adds [client certificate authentication](https://webapi.arcus-azure.net/features/security/auth/certificate) mechanism to the API project
   * `None`: no authentication configured on the API project.
+* `-ia|--include-appsettings` (default `false`): includes a `appsettings.json` file to the web API project.
```

Comment:
Maybe this is more clear than `-ai`?
```suggestion
* `-app|--include-appsettings` (default `false`): includes a `appsettings.json` file to the web API project.
```

Reply:
Hmm, I don't know if I can change this; comes from the help you see.

Reply:
But who determines these values? Don't we manage that?

Reply:
No, should do some research; maybe there is some config file to be added of some sorts. 
Now, it's just generated.

Reply:
Ok, resolving